tests/comfy_cli/test_global_python_install.py

```
@@ -0,0 +1,201 @@
+            for k, v in saved.items():
+                if v is not None:
+                    os.environ[k] = v
+
+    return _Ctx()
+
+
+class TestGlobalPythonDetection:
+    """ensure_workspace_python must return sys.executable when running from the
+    system Python and must NOT create a .venv."""
+
+    def test_global_python_skips_venv_creation(self, tmp_path):
```

Pylint (4.0.5)

[convention] Missing function or method docstring

(C0116)